app/ui/content/lib/ua.js

```diff
@@ -0,0 +1,46 @@
+/*
+ Copyright 2016 Mozilla
+
+ Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ this file except in compliance with the License. You may obtain a copy of the
+ License at http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software distributed
+ under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ specific language governing permissions and limitations under the License.
+ */
+
+// import request from '../../../ui/browser/lib/request';
+
+const request = () => {};
+
+export default {
```

Comment:
we should export each of these functions on their own line rather than nested for consistency

`export async function query() {`